library/components/TPdf.vue

```diff
@@ -27,9 +35,15 @@
             },
         },
         methods: {
-            downloadPdf() {
+            async downloadPdf() {
                 const url = this.page.url + window.location.search;
-                return this.downloadPdfFile(url);
+                this.showPdfTooltip = true;
+                await this.downloadPdfFile(url);
```

Comment:
Just out of curiosity, what does the await here accomplish?

Reply:
Ahh totally missed it, I needed it before but because it was emitting the state to parent, forgot to remove it, thanks for pointing this out @msorber .